### PR TITLE
Improve install instructions

### PR DESCRIPTION
**Pull request by jonashaag**
Inspired by https://github.com/mamba-org/mamba/issues/2907.

### CODE REVIEW

File: docs/source/user_guide/troubleshooting.rst [hunk 1]
```diff
@@ -54,6 +65,7 @@ Or in your :file:`~/.condarc` file:
   ...
   channels:
     - ...
+    - defaults  # BAD! Remove this if it exists.
```
Comment (AntoinePrv) [inline docs/source/user_guide/troubleshooting.rst:68]: What about this? It does deactivate the `defaults` altogether so maybe not the best.
```suggestion
default_channels: []
channels:
  - conda-forge
```

File: docs/source/mamba-installation.rst [hunk 2]
```diff
@@ -7,30 +7,30 @@ Mamba Installation
...
    This way of installing Mamba is **not recommended**.
-   We strongly recommend to use the Mambaforge method (see above).
+   We strongly recommend to use the Miniforge method (see above).
 
 To get ``mamba``, just install it *into the base environment* from the ``conda-forge`` channel:
 
 .. code:: bash
 
-   # NOT RECOMMENDED: This method of installation is not recommended, prefer Mambaforge instead (see above)
+   # NOT RECOMMENDED: This method of installation is not recommended, prefer Miniforge instead (see above)
```
Comment (AntoinePrv) [inline docs/source/mamba-installation.rst:33]: TIL

**Review by AntoinePrv — APPROVED** [review]

**Comment by jonashaag** [comment]
My docs changes don't seem to be published?

**Comment by JohanMabille** [comment]
Looks like the build on RTD is broken

**Pull request merged**